test/napi/src/js/extract.rs

```diff
@@ -22,9 +22,9 @@ pub fn extract_values(mut cx: FunctionContext) -> JsResult<JsArray> {
         String,
         Date,
         Handle<JsValue>,
-        ArrayBuffer,
+        ArrayBuffer<Vec<u8>>,
```

Comment:
Need a test or two for the typed array extractors. We could use `String` or `Cow` as an example that couldn't otherwise use the shorthand. A more common example is `bytes::Bytes`, but that would require an additional dev dependency.

Reply:
Added.